Makefile

```diff
@@ -233,10 +233,10 @@ serve-r:
 
 # Build the _shinylive directory for deployment of both R and Python sites
 _shinylive:
-	$(MAKE) buildjs-prod
-	cp -Lr $(SITE_DIR) $(SHINYLIVE_DIR)/py
 	$(MAKE) buildjs-prod-r
-	cp -Lr $(SITE_DIR) $(SHINYLIVE_DIR)/r
+	cp -Lr $(SITE_DIR)/. $(SHINYLIVE_DIR)/r
+	$(MAKE) buildjs-prod
+	cp -Lr $(SITE_DIR)/. $(SHINYLIVE_DIR)/py
```

Comment:
Just making sure: the order of these builds doesn't matter anymore, right?

Reply:
Correct, the order change is just left over from earlier experimentation. I should flip it back to make the intention clearer here.

The addition of `/.` to the end of the path is important, though. Without it you end up with `$(SHINYLIVE_DIR)/r/r` if `$(SHINYLIVE_DIR)/r` already exists.